pkg/cli/mirror/catalog_images.go

```diff
@@ -316,7 +341,18 @@ func layerFromFile(targetPath, path string) (v1.Layer, error) {
 	return tarball.LayerFromReader(&b)
 }
 
-func runDebug(cmd *exec.Cmd) error {
-	logrus.Debugf("command: %s", strings.Join(cmd.Args, " "))
-	return cmd.Run()
+func (o *MirrorOptions) getRemoteOpts(ctx context.Context) (options []remote.Option) {
+	return append(
+		options,
+		remote.WithAuthFromKeychain(authn.DefaultKeychain),
+		remote.WithContext(ctx),
+		remote.WithTransport(o.createRT()),
+	)
+}
+
+func (o *MirrorOptions) createRT() http.RoundTripper {
+	transport := http.DefaultTransport.(*http.Transport).Clone()
+	transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: o.DestSkipTLS}
+	var rt http.RoundTripper = transport
+	return rt
```

Comment:
You should be able to return transport directly
```suggestion
	return transport
```